### ISSUE TITLE
Amygdala microinfarct

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7057]
**Preferred term label:**
Amygdala microinfarct

**Synonyms**

**Definition (free text, please give PubMed ID)**
An infarct located in the almond-shaped body of basal nuclei anterior to the inferior horn of the lateral ventricle of the brain, within the temporal lobe that can only be observed microscopically.
PMID:10867785

**Parent term (use hpo.jax.org/app)**
Abnormal amygdala morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 606889

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added term, thank you @sarahstofel

**Comment by pnrobinson** [comment]
@sarahstofel  PS, I did not see a reference to this feature in the OMIM entry. If you have a PMID for an article that describes this please reopen this issue and we will add the annotation.

**Comment by sbello** [comment]
Hi Peter,
The reference attached to the definition is the source for the phenotype association to Alzheimer's disease
